Jenkinsfile

```diff
@@ -288,7 +288,7 @@ def cmake_build(Map conf=[:]){
     if(!setup_args.contains("NO_CK_BUILD")){
         if (setup_args.contains("gfx90a") && params.NINJA_BUILD_TRACE){
             echo "running ninja build trace"
-            setup_cmd = conf.get("setup_cmd", "${cmake_envs} cmake -G Ninja ${setup_args}   .. ")
+            setup_cmd = conf.get("setup_cmd", """${cmake_envs} cmake -G Ninja ${setup_args} -DCMAKE_CXX_FLAGS=" -O3 -ftime-trace "  .. """)
```

Comment:
`-O3` should be set somewhere else? meaning not needed here

Reply:
Also, are we sure here there are no build (and json) files produced by the previous builds when we invoke the build command?

Reply:
Clang currently uses -O2 as default, we want to use -O3, so we have to set it explicitly.
We invoke this tracing in the daily builds where we don't use sccache and build from scratch.